0005-onboarding-process.md

```diff
@@ -0,0 +1,63 @@
+# Decide on an onboarding process to unblock new volunteer involvement
+
+* Status: proposed 
+* Deciders: @Davidezrajay @sebastiangaertner @ZavenArra Tabitha Kelly
+* Date: 2022/05/22
+
+
+## Context and Problem Statement
+
+Greenstand sources engineering and operations volunteers from the public, and currently there is no onboarding process in place.  Past onboarding processes required more human resources hours than the organization currently has available, but did produce documentation that could be leveraged in a more automated process.  
+
+## Decision Drivers <!-- optional -->
+
+* Ease of implementation
```

Comment:
and maintenance

Reply:
Incorporated process maintenance into decision drivers.